MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -16,8 +16,14 @@ public partial class GraphicsDevice : IDisposable
         private bool _isDisposed;
 
         private BlendState _blendState;
-        private BlendState _actualBlendState;
-        private DepthStencilState _depthStencilState = DepthStencilState.Default;
+        private BlendState _actualBlendState;
```

Comment:
Do you mind cleaning up the blend state declarations?  Like here you have the `_blendState` and `_actualBlendState`, but have all the `_blendStateAdditive`/`_blendStateAlphaBlend`/etc down around line 300.